### DISCUSSION TITLE
Fantastic product - best way to support?

### DISCUSSION THREAD

**Discussion by chrisk-7777** [tinkerun/tinkerun#9] [General]
Even though this product hasn't reached its first major release (at the time of writing, its 0.1.5), it is heads and shoulders above Tinkerwell. 

How can I as an end user best support it? Bug reports? PR's? Docs? Etc.

**Comment by billyct** [comment]
cool, Bug reports, PR's, Docs, etc, all are welcome.

and also if you have any idea of the product features, please let me know.
